src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double MULTIPLIER = 1.5;
+    private Object[] elementsData;
+    private int size;
+
+    public ArrayList() {
+        this.elementsData = (T[]) new Object[DEFAULT_SIZE];
+    }
+
     @Override
     public void add(T value) {
-
+        createNewArray();
```

Comment:
The problem here is in method name. It's still not a good name for this method. It's confusing to understand by method's name: what does this method do. We have already created new array in constructor, why do we need to create it again?

You can fix it in two ways:
1) You can name this method like: "checkSizeAndResize()". 
2) But better way is to take "IF" out of resize(): 
if (size == elementsData.length) {
    resize();
}